bitsandbytes/functional.py

```diff
@@ -1798,7 +1891,7 @@ def igemmlt(A, B, SA, SB, out=None, Sout=None, dtype=torch.int32):
 
 def mm_dequant(
     A,
-    quant_state,
```

Comment:
Same as above. Althought, it is very unlikely that a 3rd party code uses this function, its better to leave the name as `state` to ensure backward compatibility of 3rd party code.

Reply:
done